readme.md

```diff
@@ -176,6 +178,23 @@ the anecdote, you can throw away the data."
 
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
+## Pyschological Disorders
+
+## Trichotillomania
```

Comment:
This should be `#### [Tricho...](link-to-wikipedia)`

Reply:
Done that :sweat_smile: